controller/src/bin/xcvradm.rs

```diff
@@ -1379,109 +1999,210 @@ fn display_list<T: std::fmt::Display>(items: impl Iterator<Item = T>) -> String
-        } else {
-            println!("  {:>NAME_WIDTH$}: {unsupported}", "Aux 1");
+
+                // Print each auxiliary monitor.
+                //
+                // The requires that we print the "observable", the thing being measured
+                // as well. Each line is formatted like:
+                //
+                // Aux N, <observable> (<units>): <value>
+                let (aux1_name, aux1_value) =
+                    aux1_monitor_value(monitor.aux_monitors.and_then(|aux| aux.aux1));
+                if let Some(aux1_value) = aux1_value {
+                    println!("  {aux1_name:>NAME_WIDTH$}: {aux1_value}");
+                } else {
+                    println!("  {:>NAME_WIDTH$}: {unsupported}", aux1_name);
+                }
+
+                let (aux2_name, aux2_value) =
+                    aux2_monitor_value(monitor.aux_monitors.and_then(|aux| aux.aux2));
+                if let Some(aux2_value) = aux2_value {
+                    println!("  {aux2_name:>NAME_WIDTH$}: {aux2_value}");
+                } else {
+                    println!("  {:>NAME_WIDTH$}: {unsupported}", aux2_name);
+                }
+
+                let (aux3_name, aux3_value) =
+                    aux3_monitor_value(monitor.aux_monitors.and_then(|aux| aux.aux3));
+                if let Some(aux3_value) = aux3_value {
+                    println!("  {aux3_name:>NAME_WIDTH$}: {aux3_value}");
+                } else {
+                    println!("  {:>NAME_WIDTH$}: {unsupported}", aux3_name);
+                }
+
+                // Print additional newline between each port for clarity.
+                need_newline = true;
+            }
+            OutputKind::Parseable {
+                fields, separator, ..
+            } => {
+                if fields.len() > 0 {
+                    println!(
+                        "{}",
+                        fields
+                            .iter()
+                            .map(|field| match field {
+                                MonitorFields::Port => port.to_string(),
+                                MonitorFields::Temperature => monitor
+                                    .temperature
+                                    .map(|t| t.to_string())
+                                    .unwrap_or_else(|| String::from("unsupported")),
+                                MonitorFields::SupplyVoltage => monitor
+                                    .supply_voltage
+                                    .map(|v| v.to_string())
+                                    .unwrap_or_else(|| String::from("unsupported")),
+                                MonitorFields::AverageRxPower => monitor
+                                    .receiver_power
+                                    .as_ref()
+                                    .map(|rx| {
+                                        format!("[{}]", display_list(rx.iter().map(|x| x.value())))
```

Comment:
I might remove the `[]` surrounding this, just like you did for the parseable output of the memory `read` subcommand. Just one less thing to ignore in a parsing program. Same for the other array values, TX bias and power.